app.py

```diff
@@ -1,22 +1,76 @@
+import os
+import re
+import string
 import config
-from os import urandom
-from core.helpers import initialize
+
+
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
+from flask_graphql import GraphQLView
+from flask_graphql_auth import GraphQLAuth
+
+from dvga.helpers import initialize
+from dvga.helpers import level_is_easy
+from dvga.helpers import level_is_hard
+
+from secrets import choice
+
 
 app = Flask(__name__, static_folder="static/")
-app.secret_key = urandom(24)
+app.schema = None
+chrset = string.ascii_lowercase + string.digits
+
+# Weakened deliberately to allow for JWT bruteforce 
+app.secret_key = ''.join([choice(chrset) for _ in range(6)]) 
 app.config["SQLALCHEMY_DATABASE_URI"] = config.SQLALCHEMY_DATABASE_URI
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = config.SQLALCHEMY_TRACK_MODIFICATIONS
 app.config["UPLOAD_FOLDER"] = config.WEB_UPLOADDIR
 
 db = SQLAlchemy(app)
+auth = GraphQLAuth(app)
```

Comment:
is there a reason that this is defined twice? L30 and L40 I believe...

Reply:
That's a mistake. I should remove it.